### DISCUSSION TITLE
SelectOneMenu custom filter problems

### DISCUSSION THREAD

**Discussion by Just-Dominic** [primefaces/community#1302] [PrimeFaces]
Hi,

I have a selectOneMenu with 2 checkboxes (selectManyCheckbox) to filter the list. The boxes are in the footer facet (and styled tot the top) of the selectOneMenu. I've got it working but after the update the panel is re-opening again. Is there a way to keep the panel open and just update the values of the list when I use the checkboxes? I can't update the table inside the panel I reckon? 

Video
https://github.com/primefaces/community/assets/144908012/70fa0cc3-5fea-48b6-adb1-3c07ae50afff

Here is the xhtml
[Code.txt](https://github.com/primefaces/community/files/14418627/Code.txt)

Dominic

**Comment by melloware** [comment]
Please provide an executable example using the [PrimeFaces Test](https://github.com/primefaces/primefaces-test) project.  It is the only way developers can debug your problem to help.

**Comment by Just-Dominic** [comment]
We are not allowed to share data with you. Is the code and the video not enough?

**Reply by melloware** [reply]
No I can't "debug" a video.

**Comment by Just-Dominic** [comment]
You can delete this discussion then :(
